src/main/java/com/smi/xxx/rest/base/JwtFilter.java

```diff
@@ -28,7 +29,7 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
     } else {
       response.setStatus(HttpServletResponse.SC_UNAUTHORIZED);
 
-      return;
+      // return; sonarlint
```

Comment:
sonarlint flagged this return as a code smell.

Unsure if return is a must here to stop the request lifecycle